event-handlers/src/main/java/no/sikt/nva/nvi/events/evaluator/calculator/CreatorVerificationUtil.java

```diff
@@ -73,6 +96,10 @@ private static URI extractContributorId(JsonNode creatorNode) {
         return URI.create(extractJsonNodeTextValue(creatorNode, JSON_POINTER_IDENTITY_ID));
     }
 
+    private static String extractContributorName(JsonNode creatorNode) {
```

Comment:
Maybe input parameter named as in another method which consumes contributor node - contributorNode